source/test/F0.Minesweeper.Components.Tests/CellTests.cs

```diff
@@ -0,0 +1,146 @@
+using System;
+using Bunit;
+using Bunit.Rendering;
+using F0.Minesweeper.Components.Abstractions;
+using F0.Minesweeper.Components.Abstractions.Enums;
+using F0.Minesweeper.Logic.Abstractions;
+using FluentAssertions;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using Xunit;
+
+namespace F0.Minesweeper.Components.Tests
+{
+	public class CellTests : TestContext
+	{
+		Mock<ICellStatusManager> cellStatusManagerMock;
+
+		public CellTests()
+		{
+			cellStatusManagerMock = new Mock<ICellStatusManager>();
```

Comment:
I prefer a "Strict"-Mock, which we then "Verify" in the `Dispose`.

Reply:
Resolved but rather went with `Mock.VerifyAll` since I personally prefer that

Reply:
cool